frontend/src/Rankings.tsx

```diff
@@ -165,16 +161,16 @@ function Rankings({
                             <TableHeaderCell level={0} rowSpan={3}>
                                 Games
                             </TableHeaderCell>
-                            <TableHeaderCell level={0} rowSpan={3}>
-                                Games {year}
-                            </TableHeaderCell>
-                            <TableHeaderCell level={0} colSpan={6}>
-                                Base Game {year}
+                            <TableHeaderCell level={0} colSpan={7}>
+                                Yearly Stats {year}
                             </TableHeaderCell>
                         </TableHeaderRow>
 
                         <TableHeaderRow>
                             {/* Base */}
+                            <TableHeaderCell level={2} rowSpan={2}>
```

Comment:
`level={2}` needs to be `level={1}` 😬

![image](https://github.com/user-attachments/assets/392bd35c-4608-4dab-ab12-2ec7026926a9)

Hopefully when I move us away from MUI tables, I can make the headers less finicky.

Reply:
God forbid you let that lil header live out its dreams of being part of the data. Jeez

Reply:
😂😂